app/components/ButtonGroup/ButtonGroup.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function ButtonGroup({ buttons, }) {
+  const style = {
+    height: `${100 / buttons.length}%`,
+  };
+
+  return (
+    <div className="buttonGroup">
+      {
+        buttons.map((button, index) => {
+          const {
+            text,
+            classes,
+            disabled,
+            onClick,
+          } = button;
+          const buttonProps = {
+            type: 'button',
+            key: `button${index}`,
+            className: classes,
+            style,
+            onClick,
+            disabled,
+          };
+          return (
+            <button { ...buttonProps }>
+              {text}
```

Comment:
Now that your props are in an object, you can learn about the `children` prop and compose it right in with the rest:
```js
const buttonProps = {
  type: 'button',
  // the rest...,
  children: text,
}
return <button {...buttonProps} />
```

Reply:
learned about it, but haven't grasped fully the intricate ways of implementing it fully yet :)

done